ios/MeasureSDK/MeasureSDKTests/SessionManagerTests.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  SessionManagerTests.swift
+//  MeasureSDKTests
+//
+//  Created by Adwin Ross on 02/09/24.
+//
+
+import XCTest
+@testable import MeasureSDK
+
+final class SessionManagerTests: XCTestCase {
```

Comment:
Add a test for start method covering both cases, where we're able to get the session ID and one where the start method throws as the session wasn't created yet.